components/pipeline-service/base/external-secrets/kustomization.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - tekton-results
-  - pipelines-as-code
+  # - pipelines-as-code
```

Comment:
don't remove this now;

pipeline service team will remove it remove it when we remove the pipelines-as-code namespace

Reply:
un-commented this.